status/k8s.go

```diff
@@ -357,9 +358,18 @@ func (k *K8sStatusCollector) Status(ctx context.Context) (*Status, error) {
 		}
 		if !k.statusIsReady(s) && k.params.Wait {
 			time.Sleep(defaults.WaitRetryInterval)
+			statusFmt := s.Format()
+			for i := 1; i < lines; i++ {
```

Comment:
Wondering if we could use a progress bar for this. Likely there might be other places that could benefit with user feedback.

We should probably use a help func to clear out the output if any.
nit: Docs might help here to indicate what this sequence does...

Reply:
Ah yes, the sequence is to set the cursor one line up.

Reply:
As for the progress bar, I'm not a big fan because:

1.  we can't really know the % progress
2. as a user, I'd rather have details of what is going on (and see if something is stuck) in the same format as `cilium status`)